File: C/hello.c
#include <stdio.h>

int main()
{
	char usrname[20];
	int a, b;
	int c;

	printf("\nWhat's your name?: ");
	scanf("%19s", &usrname);
	printf("\nWhy hello, %s!", usrname);
	
	a = 2;
	b = 2;
	c = a + b;
	printf("\n2 plus 2 is: %d\n", c);

	return(0);
}

File: Haskell/hello.hs
word :: String 
word = "world"

a,b,c :: Int
a = 2
b = 2
c = a + b

main = do 
	putStrLn "What's your name? "
	usrname <- getLine
	print "Hello"	
	putStrLn ("Hello " ++ usrname)
	putStrLn "2 plus 2 is: "
	print c

